File: jvm2/src/Model/Player/Units/PBuilder.kt
package Model.Player.Units

import Model.Location
import Model.Move
import Model.Neutral.Units.Builder

class PBuilder(location: Location) : Builder() {
    override var loc = location
    override var use = true
    override var drive = true
    override var owner = 1
    override var icon = "pB"
}

File: dotnet/src/Model/Player/Units/PBuilder.cs
using System;

namespace PlanetCS1 {
    class PBuilder : Builder {
        public PBuilder(Location location) {
            this.loc = location;
            this.use = true;
            this.drive = true;
            this.owner = 1;
            this.icon = "pB";
        }
    }
}